libraries/Microsoft.Bot.Schema/ConversationReference.cs

```diff
@@ -89,7 +90,12 @@ public ConversationReference(string activityId = default(string), ChannelAccount
         /// referenced conversation may be performed
         /// </summary>
         [JsonProperty(PropertyName = "serviceUrl")]
-        public string ServiceUrl { get; set; }
-
+        public string ServiceUrl { get; set; }    
+        
+        /// <summary>
```

Comment:
these files are autogenerated based on the swagger definition, so any changes here should be handled automatically.